runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ArtifactServiceStager.java

```diff
@@ -57,6 +54,9 @@
 import org.apache.beam.model.jobmanagement.v1.ArtifactStagingServiceGrpc.ArtifactStagingServiceStub;
 import org.apache.beam.sdk.util.MoreFutures;
 import org.apache.beam.sdk.util.ThrowingSupplier;
+import org.apache.beam.vendor.grpc.v1.io.grpc.Channel;
```

Comment:
Code style bikeshed: Would it be useful to separate out vendored imports? For checkstyle we could declare a separate import group to sort them at the end: http://checkstyle.sourceforge.net/config_imports.html#ImportOrder

Reply:
Makes sense to me, thanks